build.sbt

```diff
@@ -33,7 +33,7 @@ libraryDependencies ++= Seq(
   "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % sttpVersion,
   "uk.gov.nationalarchives" %% "tdr-graphql-client" % "0.0.94",
   "uk.gov.nationalarchives" %% "tdr-auth-utils" % "0.0.120",
-  "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.313",
+  "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.314",
```

Comment:
Is that the latest version of the graphql?

Reply:
Updated with the latest version.